### PR TITLE
add action

### PR DESCRIPTION
**Pull request by yossale**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: 'Altostra Login'
+description: 'GitHub Action to login against an Altostra account'
```
Comment (yevk) [inline action.yml]:
```suggestion
description: 'Log in to Altostra account.'
```

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+author: 'Altostra'
+branding:
+  icon: 'triangle'
+  color: 'purple'
```
Comment (yevk) [inline action.yml:6]: What does this mean?

File: action.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+inputs:
+  api-token:  # id of input
+    description: 'Your Altostra API token from https://app.altostra.com/settings/tokens'
```
Comment (yevk) [inline action.yml]:
```suggestion
    description: 'An Altostra API token from https://app.altostra.com/settings/tokens'
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,32 @@
-# altostra-login-action
+![altostra-integration](./resources/img/github-actions-ci-cd-altostra.png)
+## About
+
 GitHub Action to login against an Altostra account.
```
Comment (yevk) [inline README.md]:
```suggestion
Use this Action to log into your Altostra account and store credentials for later use with other Altostra Actions.
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,32 @@
...
+  hello_world_job:
+    runs-on: ubuntu-latest
+    name: A basic Altostra job
+    steps:
+    - uses: actions/checkout@v2
+    - id: Login
+      uses: altostra/login-action@v1
+      with:
+        api-token: "${{ secrets.ALTO_API_TOKEN }}"
+```
```
Comment (yevk) [inline README.md:37]: Add another step with the deploy command to give the reader a taste of an actual flow.

**Review by yevk — CHANGES_REQUESTED** [review]

File: action.yml (hunk 2 above)
Comment (yossale) [reply action.yml:6]: The little signs at the side of the action. Still haven't got it to work , but it's per their documentation :)

**Review by yevk — APPROVED** [review]

**Pull request merged**